src/public/interfaces.ts

```diff
@@ -394,3 +394,20 @@ export interface OpenConversationRequest {
   */
   onCloseConversation?: (subEntityId: string, conversationId?: string) => void;
 }
+
+export interface IAppLoadFailReason {
+  reason: AppLoadFailReasonTypes;
+  message?: string;
+}
+
+export interface IAppLoadEvent {
```

Comment:
NIT: I think `IAppInitEvent` or `IAppInitializationEvent` might be a better name for this